console/client/src/features/verbs/VerbTab.tsx

```diff
@@ -23,7 +24,7 @@ export const VerbTab: React.FC<Props> = ({ id }) => {
   }
 
   return (
-    <div className='min-w-0 flex-auto p-4'>
+    <div className='min-w-0 flex-auto'>
```

Comment:
Do we want this change here? It looks like it makes the verb tab have no margin

<img width="769" alt="Screenshot 2023-08-24 at 8 38 15 PM" src="https://github.com/TBD54566975/ftl/assets/51647/5b422d35-4bc9-4624-914c-f6074bb78036">

Reply:
Looks like padding it got dropped when I pulled in commits from main: https://github.com/TBD54566975/ftl/pull/320/commits/1a397f9aefaf1f1ba1d1dc357d57027da3911aa4#diff-7a08d7a5e05b5233e6546527ec691b6514a387909f944a33ab3e92df31dc680eL130

 Also yeah I mean to rip that off. We should apply consistent padding to all panels not just the verb panel.

Reply:
Added the padding back the was missing and adjusted padding on verb form selector

Reply:
Yup! I'm very open to consistent padding here. If you have ideas I'd love to see some options. For this example, we don't pad all the panels the same because some of the have edge to edge components, like the filter bar on the timeline. So we don't pad the panel and let the embedded component control it's padding.